src/main/resources/META-INF/resources/primefaces/menu/menu.css

```diff
@@ -203,6 +203,13 @@
     border-top: 0px none;
 }
 
+body .ui-menu .ui-divider.ui-divider-vertical {
+    display: inline-block;
+    padding: 1rem 0;
+    margin: 0;
+    top: 10px;
```

Comment:
This is the part I am not sure about had to make top 10px for the divider to be centered in the menu else it did not look right.